### PR TITLE
PGP encryption

### PR DESCRIPTION
**Pull request by catlook**
In a comment on the pull request, include the following information and check off the list.

 - [ ] Mini bio
  - Soft capped to 75 words
 - [ ] Desired date of publishing
  - Make sure this matches the `date` in the front matter
 - [ ] Social media contact info
 - [ ] Redaction: (yes|no)
  - Allow redaction, editing or structure changes by technical writer staff
 - [ ] Comments: (yes|no)
  - If you're willing to answer comments about your post, set `comments: true`.
  - Make sure to subscribe to your post comments by clicking on the **Subscribe** link at the very bottom of the post _once_ it's published.

@catlook Please review.

### CODE REVIEW

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-8f3101509a/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-3c5cf82cea/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-3c5cf82cea/)

**Comment by rackernexus** [comment]
:mag_right: [Preview](https://staging.developer.rackspace.com/build-8870a0124f/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-8870a0124f/)

**Comment by rackernexus** [comment]
If you want to double-check your formatting, check [the preview](https://staging.developer.rackspace.com/build-3bc0dc406f/).

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-718c19b0fc/)

**Comment by rackernexus** [comment]
HULK [PREVIEW](https://staging.developer.rackspace.com/build-6345d3e3eb/)

**Comment by rackernexus** [comment]
:eyes: [Preview](https://staging.developer.rackspace.com/build-cc8866a0fe/)

File: _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+
+   Now, you'll be shown the key name and user id that you chose. Then you'll be
+   prompted to "Change (N)ame, (C)omment, (E)mail, or (O)kay/(Q)uit?", choose
+   **O** for okay.
+
+   The following image shows these prompts and responses:
+
+   ![]({% asset_path 2018-09-19-introduction-to-pgp-encryption-and-decryption/Picture3.png %})
+
+   Note: You might be required to enter a passphrase to protect your secret key.
```
Comment (stephamon) [inline _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md]: Format as a call out note box

File: _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+#### Encrypt a file with a passphrase
+
+Use the following command to encrypt a file with a passphrase:
+
+     $ gpg -s --no-tty --always-trust --passphrase "passphrase@test" -u <Key_Name>-pub-sub.asc "data_file.txt"
+
+If you defined your key in the source system with a passphrase, the same
+passphrase must be used in the preceding command.  In our example,
+“passphrase@test” is the passpharse to be used.
```
Comment (stephamon) [inline _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md]: passphrase, not passpharse :)

File: _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+     $ echo $PASSPHRASE | gpg --batch --yes  --passphrase-fd 0 data_file.txt.gpg
+
+### Conclusion:
+
+PGP Encryption secures data transmission over the internet and only the person
+with whom you have shared the keys is able to decrypt the data and use it. This
+protects your data so that it cannot be misused, even if someone has the file.
+Without keys, no one can decrypt the file and access the data. If you need to
+transfer sensitive information, use file encryption functionality. These examples
```
Comment (stephamon) [inline _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md]: The examples in this blog...

File: _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+    - security
+---
+
+The GNU Privacy Guard (GPG) is a complete and free implementation of the
+OpenPGP standard as defined by RFC4880 also known as PGP (Pretty Good Privacy).
+GPG, also known as GnuPG, is a command line tool with features for easy
+integration with other applications.
+
+Although most companies exchange sensitive data like payment details, employee
+information, and so on over the internet, these companies use PGP encryption to
```
[the remark this answers is not in the document]
Comment (catlook) [reply _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md]: the companies that exchange sensitive data referenced in the opening clause

[the remark this answers is not in the document]
Comment (catlook) [reply _posts/2018-09-19-introduction-to-pgp-encryption-and-decryption.md]: i'll rephrase

**Comment by rackernexus** [comment]
Your content preview [is now ready](https://staging.developer.rackspace.com/build-ca38df734d/). :bow:

**Pull request merged**